lib/geo.js

```diff
@@ -0,0 +1,71 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Apache License 2.0.
+// License text available at https://opensource.org/licenses/Apache-2.0
+
+'use strict';
+
+var URL = require('url');
+var assert = require('assert');
+var util = require('util');
+var _ = require('lodash');
+
+module.exports = mixinGeo;
+
+function mixinGeo(CouchDB) {
```

Comment:
If this is meant for cloudant, should this be `Cloudant` instead of `CouchDB`?

Reply:
Yes, you are right. I went ahead and amended the PR.